### DISCUSSION TITLE
ISO: Fixed vs Variable

### DISCUSSION THREAD

**Discussion by tetrahydra** [gopro/labs#756] [General]
Hi guys, the thing I like about GoPro is the min/max ISO settings. I set my EXPS=1 and I noticed that the ISO jumps quite a lot.

I know this question may sound silly, but do you think it is wise to set MIN = MAX = single value?

Another thing, does GoPro (specifically the GP11 sensor) have a native ISO, like Sony, where it has the lowest noise? I am using "Linear" which crops into the sensor and upscales to 5.3K. Currently, I set my ISO to 400 and WB to 5000. I am thinking of raising the ISO to 800 but I am a bit afraid of the noise.

I found this ISO comparison, somewhat ISO 800 is "acceptable"
https://www.youtube.com/watch?v=rt3frTm4kLs

**Comment by dnewman-gpsw** [comment]
Native ISO would be 100.  Lowest noise.
Setting an ISO MAX of 800 is common setup.
